test/external/CMakeLists.txt

```diff
@@ -7,7 +7,7 @@ include(FetchContent)
 FetchContent_Declare(
   ringtest
   GIT_REPOSITORY https://github.com/neuronsimulator/ringtest
-  GIT_TAG ee24c8
+  GIT_TAG 30cd087a8563532482815a8260a2f93a4ae9899b
```

Comment:
@WeinaJi : this now can be changed.

Reply:
Yes, I will update it with the latest commit in master